packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -565,6 +569,63 @@ class _ErrorNavigatorButton extends StatelessWidget {
   }
 }
 
+class InspectorDetailsOption extends StatelessWidget {
+  const InspectorDetailsOption({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: preferences.inspector.defaultInspectorDetailsView,
+      builder: (context, selection, _) {
+        final theme = Theme.of(context);
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              'Select the tab which opens by default when opening the inspector.',
```

Comment:
Suggestion: "Set the default tab for the inspector on opening"

Reply:
Or "Select the default tab for the inspector on opening" (mostly trying to avoid having the word "open" appear twice)